README.md

```diff
@@ -10,7 +10,7 @@ This is a Dockerfile to use [NVIDIA SDK Manager](https://docs.nvidia.com/sdk-man
 ### Download NVIDIA SDK Manager
 Please download the package of NVIDIA SDK Manager from <https://developer.nvidia.com/embedded/dlc/nv-sdk-manager>.  
 And, please put the package of NVIDIA SDK Manager in the same directory as the Dockerfile.  
-This time, I used `sdkmanager_0.9.12-4180_amd64.deb`.
+This time, I used `sdkmanager_0.9.14-4964_amd64.deb`.
 
 ### Build Docker image
```

Comment:
@graugans Thank you for your proposal! I think that your change is good.
I hope you add the procedure to specify `SDK_MANAGER_VERSION` as of `docker build`.

Reply:
@atinfinity sorry for the late response. I have not yet added advise how to set this from the docker command line. I will add this after cleaning my TODO list.

Reply:
@graugans 

> I have not yet added advise how to set this from the docker command line.

I tried to set `SDK_MANAGER_VERSION` using `--build-arg`.  
It works well.

```shell
$ docker build --build-arg SDK_MANAGER_VERSION=1.0.0-5517 -t jetpack .
```

Reply:
I've updated the PR